packages/utilities/src/Customizer.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { BaseComponent } from './BaseComponent';
+import { assign } from './object';
+
+export interface ISettings {
```

Comment:
I'd like to check this in as an initial implementation, but will add examples and refine this in a separate PR.